src/inc/msquic.h

```diff
@@ -272,6 +272,7 @@ typedef enum QUIC_EXECUTION_CONFIG_FLAGS {
     QUIC_EXECUTION_CONFIG_FLAG_XDP              = 0x0004,
     QUIC_EXECUTION_CONFIG_FLAG_NO_IDEAL_PROC    = 0x0008,
     QUIC_EXECUTION_CONFIG_FLAG_HIGH_PRIORITY    = 0x0010,
+    QUIC_EXECUTION_CONFIG_FLAG_AFFINITIZE       = 0x0020,
```

Comment:
IMO, this should indicate what the threads are being affinitized to - NUMA node, single RSS CPU, something else?

Reply:
I'd prefer to just leverage documentation for what, which is currently being deferred since these are all prototype features. I just want to get this in so our partners can experiment. If it doesn't actually help, it might ultimately be removed.

Reply:
Generally speaking, I think even experimental features should be designed as if they will be retained, because it is difficult to remove/rename existing dependencies. That said, I'm ok with unblocking folks here.